src/pages/Help/index.js

```diff
@@ -30,11 +32,15 @@ const Game = () => (
   <ContentFromRTD page="protein_families_game.rst" format="faq" />
 );
 
+// FlowFix me comment needed for async components
 const Documentation = loadable({
   loader: () =>
     import(
-      /* webpackChunkName: "help-documentation" */ 'components/Help/Documentation'
+      /* webpackChunkName: "help-documentation" */
```

Comment:
Not sure if webpack will be able to identify the chunk name if the line is split, rather use a `// prettier-ignore` for the block